os/drivers/compression/compress.c

```diff
@@ -72,6 +80,26 @@ static ssize_t comp_write(FAR struct file *filep, FAR const char *buffer, size_t
  * 	 COMPIOC_DECOMPRESS - for decompression of input data
  * 	 COMPIOC_GET_COMP_TYPE - for getting compression type
  * 	 COMPIOC_GET_COMP_NAME - for getting compression type in string
+ *   COMPIOC_FCOMP_INIT - for init the decompression framework in kernel
+ *   COMPIOC_FCOMP_GET_BUFSIZE - for getting the buffer size require to hold decompressed data
+ *   COMPIOC_FCOMP_DECOMPRESS - for getting decompressed data in buffer
+ *   COMPIOC_FCOMP_DEINIT - for cleaning the kernel decompression framework
+ * 
+ *   App shall use the ioctls as follows to perform the decompression.
```

Comment:
Add this line in the commit title also. ` App shall use the ioctls as follows to perform the decompression.`